pkg/crc/validation/validation.go

```diff
@@ -77,10 +77,13 @@ func ValidateBundlePath(bundlePath string, preset crcpreset.Preset) error {
 
 func ValidateBundle(bundlePath string, preset crcpreset.Preset) error {
 	bundleName := filepath.Base(bundlePath)
-	_, err := bundle.Get(bundleName)
+	bundleMetadata, err := bundle.Get(bundleName)
 	if err != nil {
 		return ValidateBundlePath(bundlePath, preset)
 	}
+	if bundleMetadata.GetBundleName() != bundle.GetBundleNameWithoutExtension(constants.GetDefaultBundle(preset)) {
+		logging.Warnf("Using %s bundle, but %s is expected for this release", bundleName, constants.GetDefaultBundle(preset))
+	}
```

Comment:
> Why is this duplicating code in `ValidateBundlePath` but not taking into account custom bundles?

@cfergeau it is duplicate because when bundle is extracted then `ValidateBundlePath` not even run so the warning is ignored but now it is not. This warning works with custom bundle also but it will show same message then custom bundle one, for `start` command do we really required to put custom bundle warning? May be I just put those warnings to a helper function and consume it for both functions?


> Why is the validation for `-b` different between the start and setup commands?

I don't know, may be we are assuming that during setup we always use the bundle which is not extracted so why get the bundle metadata logic?

Reply:
> May be I just put those warnings to a helper function and consume it for both functions?

Yes, this was going to be my suggestion, sharing common code is better than partially duplicating code with no clear reason for the difference in behaviour.